DEVELOPMENT.md

```diff
@@ -0,0 +1,122 @@
+<div align="center">
+
+![Uncharted Facets](assets/logo_320.png)
+
+</div>
+
+# Development Notes for Uncharted's Facets
+---  
+The Facets library is split in five packages:
+1. `facets-core` contains all the components necessary to start using facets in your project
+2. `facets-plugins` is a collection of components that extend facets' behavior, look and feel
+3. `facets-docs` documentation and examples live here
+4. `facets-builder` tool to create and customize facets (under construction, Facets 4+ feature)
+
+The only packages published to the registry are `facets-core` and `facets-plugins` 
+
+
+## Installation
+
+Use `yarn` to install the desired packages:
+```shell script
+$ yarn add @uncharted.software/facets-core
+$ yarn add @uncharted.software/facets-plugins
+```
+or `npm`:
+```shell script
+$ npm install @uncharted.software/facets-core
+$ npm install @uncharted.software/facets-plugins
+```
+
+
+## Usage
+
+The documentation is not hosted anywhere at the moment but can be accessed by building the project and running the
```

Comment:
I think this is untrue, right? Isn't it live at https://unchartedsoftware.github.io/facets/basic.html ?

Reply:
Fixed